firmware/hw_layer/backup_ram.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file	backup_ram.cpp
+ * @brief	Non-volatile backup-RAM registers support
+ *
+ * @date Dec 19, 2017
+ */
+
+#ifndef BACKUP_RAM_H_
+#define BACKUP_RAM_H_
+
+#include "main.h"
+#include "efiGpio.h"
+
+typedef enum {
+	/**
+	 * IAC Stepper motor position, 16-bit (stored in BKP0R 0..15)
+	 * Used in stepper.cpp
+	 */
+	BACKUP_STEPPER_POS,
+	/**
+	 * Ignition switch counter, 8-bit (stored in BKP0R 16..23)
```

Comment:
I do not understand the purpose of this data point. what is ignition switch?

Reply:
https://en.wikipedia.org/wiki/Ignition_switch
The counter stores the number of times the ignition switch is turned on. 
See startPrimeInjectionPulse() in main_trigger_callback.cpp, it has comments on what's going on...